Felienne/hedy
# Suppress error when sending email fails because of no aws credentials during local development #1163

**Description**

Contains the minimal changes to avoid local development failures caused by no credentials for sending emails.

**Fix for**

#1163

**How to test**

1. Run hedy locally and sign up as a new user. The sign-up call should be successful since now the email part is skipped.
2. Sign up on alpha as a new user. The sign-up call should be successful and an email should be sent.
